packages/network-controller/src/NetworkController.ts

```diff
@@ -435,12 +435,30 @@ export type NetworkControllerGetEIP1559CompatibilityAction = {
   handler: NetworkController['getEIP1559Compatibility'];
 };
 
+export type NetworkControllerFindNetworkClientIdByChainIdAction = {
+  type: `NetworkController:findNetworkClientIdByChainId`;
+  handler: NetworkController['findNetworkClientIdByChainId'];
+};
+
+export type NetworkControllerSetProviderTypeAction = {
+  type: `NetworkController:setProviderType`;
+  handler: NetworkController['setProviderType'];
+};
+
+export type NetworkControllerSetActiveNetworkAction = {
+  type: `NetworkController:setActiveNetwork`;
+  handler: NetworkController['setActiveNetwork'];
+};
+
```

Comment:
can we add basic specs for these actions?

Reply:
The methods themselves are already covered by 
 - [setActiveNetwork ](https://github.com/MetaMask/core/blob/main/packages/network-controller/tests/NetworkController.test.ts#L2851)
 - [setProviderType](https://github.com/MetaMask/core/blob/main/packages/network-controller/tests/NetworkController.test.ts#L2683)
 - [findNetworkClientIdByChainId](https://github.com/MetaMask/core/blob/main/packages/network-controller/tests/NetworkController.test.ts#L1106C13-L1106C46)
 
What would testing the messenger-equivalent look like do you think? Just copy the most basic test for each of those methods and have it use the messenger interface instead? I suppose that might catch issues with the messenger typing potentially.